internal/elixir/identify.go

```diff
@@ -0,0 +1,44 @@
+package elixir
+
+import (
+	"github.com/spf13/afero"
+
+	"github.com/zeabur/zbpack/internal/utils"
+	"github.com/zeabur/zbpack/pkg/plan"
+	"github.com/zeabur/zbpack/pkg/types"
+)
+
+type identify struct{}
+
+// NewIdentifier returns a new Elixir identifier.
+func NewIdentifier() plan.Identifier {
+	return &identify{}
+}
+
+func (i *identify) PlanType() types.PlanType {
+	return types.PlanTypeElixir
+}
+
+// Match returns true if mix.exs is found in the source
+func (i *identify) Match(fs afero.Fs) bool {
+	return utils.HasFile(fs, "mix.exs")
+}
+
+func (i *identify) PlanMeta(options plan.NewPlannerOptions) types.PlanMeta {
+	ElixirVer, err := DetermineElixirVersion(options.Source)
+	if err != nil {
+		panic(err)
+	}
+
+	ElixirEcto, err := CheckElixirEcto(options.Source)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Better to not `panic()` when something wrong. Seems like we need to re-design the API? cc @aFlyBird0